### PR TITLE
fix linting errors and add more tests

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/components/Loading/Loading.component.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import { LoadingOutlined } from '@ant-design/icons';
-import React from 'react';
 import { Spin } from 'antd';
+import React, { FC } from 'react';
 
 const antIcon = <LoadingOutlined style={{ fontSize: 24 }} spin />;
 
-const Loading = () => <Spin aria-label='loading-spinner' indicator={antIcon} />;
+const Loading: FC = () => <Spin aria-label='loading-spinner' indicator={antIcon} />;
```
Comment (rachit-aasaanjobs) [inline src/components/Loading/Loading.component.tsx]: FC should have at least one type. use FC<{}>

File: src/components/PageSidebar/PageSidebar.test.tsx [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
+import PageSidebar from './PageSidebar.component';
 import history from 'utils/misc/history';
 
 describe('PageSidebar', () => {
     it('should have link to candidates page', () => {
         const { getByLabelText } = render(
             <Router history={history}>
-                <PageSidebar location='http://localhost/candidates' match={{
+                <PageSidebar
+                  location='http://localhost/candidates'
```
Comment (rachit-aasaanjobs) [inline src/components/PageSidebar/PageSidebar.test.tsx:12]: don't use localhost anywhere in testing. wee will create a config file later in development.

File: src/components/Loading/Loading.component.tsx (hunk 1 above)
Comment (ghost) [reply src/components/Loading/Loading.component.tsx]: Yup added!

File: src/components/PageSidebar/PageSidebar.test.tsx (hunk 2 above)
Comment (ghost) [reply src/components/PageSidebar/PageSidebar.test.tsx:12]: Sure should replace this with some other url?

**Review by rachit-aasaanjobs — APPROVED** [review]

**Pull request merged**